src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -162,7 +162,7 @@ def _validate_ip_configuration(namespace, lab_vnet=None):
             # Default to shared ip configuration based on os type only if inbound nat rules exist on the
             # shared configuration of the selected lab's virtual network
             if lab_vnet.subnet_overrides and lab_vnet.subnet_overrides[0].shared_public_ip_address_configuration and \
-                    lab_vnet.subnet_overrides[0].shared_public_ip_address_configuration.inbound_nat_rules:
+                    lab_vnet.subnet_overrides[0].shared_public_ip_address_configuration.allowed_ports:
```

Comment:
There is a race in which PR makes it int first : https://github.com/Azure/azure-cli/pull/2707